adrs/2131725034/README.md

```diff
@@ -0,0 +1,37 @@
+# React Component and Hooks Props Naming
+
+* **State:** Reviewing
+* **Created:** 2022-01-24
+* **Tags:** react
+
+## Context
+
+When it comes to React, they are two categories of properties:
+
+* Data (Information)
+* Callbacks (Events, Control Flow)
+
+Data properties are primarily used for displaying information or transforming
+into another set of data.
+
+While Callbacks are used to control the flow of data such as filtering, sorting,
+for example: `onFilterAccount`, `onSortBy`, etc. Or reacting to events, the
+browser already tries to make distinction between these two categories by
+prefixing the events handlers with `on`, for example: `onClick`, `onChange`,
+etc.
+
+Understanding a React component API allows engineers to focus their attention on
+more important manners such as the business requirements.
+
+When it comes to debugging problems in production and minified code where
+finding where the problem is a challenge and time critical. Being able to reduce
+the cognitive load reading through the code base is crucial.
+
+Inconsistent properties naming promotes bikeshedding slowing down software
+development in code reviews and disagreements between developers.
```

Comment:
Might rewrite this as something like:

_Inconsistent property naming promotes bikeshedding, which slows down software development cycles in code review and results in unnecessary disagreements between developers._

also, TIL I learned bikeshedding

Reply:
Fixed it, hopefully.